sdk/storage/Azure.Storage.DataMovement/src/StreamToUriJobPart.cs

```diff
@@ -238,7 +238,8 @@ internal async Task InitialUploadCall(long blockSize, long expectedlength, bool
             {
                 if (singleCall)
                 {
-                    ReadStreamStorageResourceResult result = await _sourceResource.ReadStreamAsync().ConfigureAwait(false);
+                    ReadStreamStorageResourceResult result = await _sourceResource.ReadStreamAsync(
+                        cancellationToken: _cancellationToken).ConfigureAwait(false);
```

Comment:
This part is actually unrelated but noticed while debugging. Just adding the cancellation token to this call.